src/main/java/org/jenkinsci/plugins/codesonar/models/json/CodeSonarAnalysisWarningCount.java

```diff
@@ -0,0 +1,41 @@
+package org.jenkinsci.plugins.codesonar.models.json;
+
+/**
+ * @author aseno
+ *
+ */
+public class CodeSonarAnalysisWarningCount {
```

Comment:
The name `CodeSonarAnalysisWarningCount` suggests that this class *is* a warning count.  Evidently, this class is a JSON serialization type for a hub Chart API object.  I think it should be named accordingly, for example `CodeSonarWarningCountChartRow`.

Be sure to change variable names associated with this class.  Don't just rely on IDE refactoring/renaming function.  I think a `warningCount` variable should be a number, not an object.